### PR TITLE
INT-230: Add Create a wikia to nav (desktop only) for Japan homepage

### PR DESCRIPTION
**Pull request by perjg**
Add a button to the nav in desktop mode that links to the existing create a wikia flow (http://www.wikia.com/Special:CreateNewWiki?uselang=ja)

### CODE REVIEW

File: homepage/server/views/_partials/nav.hbs [hunk 1]
```diff
@@ -32,6 +32,13 @@
             </li>
           </ul>
         </li>
+        <li class="nav-link">
+          <div class="create-wikia">
+            <a href="http://www.wikia.com/Special:CreateNewWiki?uselang=ja" class="create-wikia-button">
+              ウィキアを作ってみよう！
```
Comment (hakubo) [inline homepage/server/views/_partials/nav.hbs]: Can we refrain from using hard-coded messages?

File: homepage/server/views/_partials/nav.hbs [hunk 2]
```diff
@@ -32,6 +32,13 @@
             </li>
           </ul>
         </li>
+        <li class="nav-link">
+          <div class="create-wikia">
```
Comment (hakubo) [inline homepage/server/views/_partials/nav.hbs]: Do you need additional did here? Can this be achieved by giving this class to a and setting display to block?

File: homepage/front/styles/component/_nav.scss [hunk 3]
```diff
@@ -34,6 +34,28 @@ header.navigation {
...
+		}
+	}
+
+	.create-wikia-button {
+		$create-wikia-button-color: #656e78;
+
+		border: solid 1px $create-wikia-button-color;
+		color: $create-wikia-button-color;
+		display: inline-block;
+		font-size: 12px;
```
Comment (hakubo) [inline homepage/front/styles/component/_nav.scss]: Can we try using rem as our base unit?

File: homepage/front/styles/component/_nav.scss [hunk 4]
```diff
@@ -34,6 +34,28 @@ header.navigation {
 		z-index: 9999;
 	}
 
+	.create-wikia {
+		display: none;
+
+		@include media ($horizontal-bar-mode) {
```
Comment (RafLeszczynski) [inline homepage/front/styles/component/_nav.scss]: `@include` should be declared before other properties

File: homepage/front/styles/component/_nav.scss [hunk 5]
```diff
@@ -34,6 +34,28 @@ header.navigation {
...
+			display: inline-block;
+		}
+	}
+
+	.create-wikia-button {
+		$create-wikia-button-color: #656e78;
```
Comment (RafLeszczynski) [inline homepage/front/styles/component/_nav.scss]: Please keep variable declaration in a single place at the top of the file

File: homepage/front/styles/component/_nav.scss [hunk 6]
```diff
@@ -34,6 +34,28 @@ header.navigation {
...
+
+		border: solid 1px $create-wikia-button-color;
+		color: $create-wikia-button-color;
+		display: inline-block;
+		font-size: 12px;
+		height: 30px;
+		line-height: 28px;
+		margin-bottom: 9px;
+		margin-top: 9px;
```
Comment (RafLeszczynski) [inline homepage/front/styles/component/_nav.scss]: if you are using the same value in multiple places please consider storing it in a variable

File: homepage/server/views/_partials/nav.hbs (hunk 2 above)
Comment (perjg) [inline homepage/server/views/_partials/nav.hbs]: @hakubo Without the extra div, the media query for hiding the button on mobile doesn't work properly.

File: homepage/front/styles/component/_nav.scss (hunk 4 above)
Comment (perjg) [inline homepage/front/styles/component/_nav.scss]: @RafLeszczynski Please see my comment in https://github.com/Wikia/mercury/pull/1536

File: homepage/server/views/_partials/nav.hbs (hunk 2 above)
Comment (RafLeszczynski) [inline homepage/server/views/_partials/nav.hbs]: @perjg why it doesn't work properly ?

Comment (perjg) [inline homepage/server/views/_partials/nav.hbs]: I tried to merge the create-wikia and create-wikia-button styles and remove the div:

```
<li class="nav-link">
  <div class="create-wikia">
    <a href="http://www.wikia.com/Special:CreateNewWiki?uselang=ja" class="create-wikia-button">
      ウィキアを作ってみよう！
    </a>
  </div>
</li>
```

The media query looks like this:

```
  display: none;

  @include media ($horizontal-bar-mode) {
    display: block;
  }
```

When I merge the styles like this, the page layout looks the same on desktop but breaks on mobile. (the button doesn't disappear).

Adding the div fixes the issue.

**Pull request merged**